output/baseoutput/clientsession.go

```diff
@@ -144,6 +150,9 @@ func (session *clientSession) processInput(maxDuration time.Duration) (chan base
 				return session.collectLeftovers(nil, endImmediately), reconnectWithDelay
 			}
 			continue
+		case <-reconnectChan:
```

Comment:
1. should have a blank line before `case` like previous `case` blocks
2. the block might be better to be moved next to `case <-maxSessionDurationSignal`, since they're meant to do the same thing. (move up one block)

Reply:
Yep, one signal will be passed to all the sessions and all of them will be triggered.